src/seedu/addressbook/commands/Command.java

```diff
@@ -67,7 +67,14 @@ public int getTargetIndex() {
         return targetIndex;
     }
 
+
+
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.